camp/sour1emon/XBox Controller Left Stick Y direction/src/main/java/frc/robot/subsystems/MotorSubsystem.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.*;
+
+import edu.wpi.first.wpilibj2.command.Subsystem;
+
+/**
+ * Represents a motor subsystem that controls a single motor.
+ */
+public class MotorSubsystem implements Subsystem {
+
+    private CANSparkMax motor;
+
+    /**
+     * Constructs a MotorSubsystem object with the specified motor port.
+     * 
+     * @param motorPort the port number of the motor
+     */
+    public MotorSubsystem(int motorPort) {
+        this.motor = new CANSparkMax(motorPort, MotorType.kBrushless);
+    }
+
+    /**
+     * Clamps a value between a minimum and maximum range.
+     *
+     * @param value The value to be clamped.
+     * @param min   The minimum value of the range.
+     * @param max   The maximum value of the range.
+     * @return The clamped value.
+     */
+    private double clamp(double value, double min, double max) {
```

Comment:
the ordering of methods in a class should be all public methods in alphabetical order followed by private methods in alphabetical order. So, this method should go last and your public methods should be in the opposite order.